test/sumo_basic_SUITE.erl

```diff
@@ -87,20 +87,38 @@ init_store(Module) ->
   sumo:persist(Module, Module:new(<<"E">>, <<"A">>, 2)),
   sumo:persist(Module, Module:new(<<"F">>, <<"E">>, 1)),
 
-  %% This is necessary to get elasticsearch in
-  %% particular to index its stuff.
-  timer:sleep(1000).
+  %% Sync Timeout.
+  %% 1. Necessary to get elasticsearch in particular to index its stuff.
+  %% 2. Necessary to Riak.
+  %% Riak clusters will retain deleted objects for some period of time
+  %% (3 seconds by default), and the MapReduce framework does not conceal
+  %% these from submitted jobs.
+  %% @see <a href="http://docs.basho.com/riak/latest/dev/advanced/mapreduce"></a>
```

Comment:
Line 96 is too long:   %% @see <a href="http://docs.basho.com/riak/latest/dev/advanced/mapreduce"></a>.